backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/versions/CassandraSchemaVersionManager.java

```diff
@@ -33,9 +33,9 @@
 import com.google.common.base.Preconditions;
 
 public class CassandraSchemaVersionManager {
-    public static final int MIN_VERSION = 1;
+    public static final int MIN_VERSION = 2;
```

Comment:
It indicates code do not support anymore schemes version 1 and dropped tge tables